app/src/main/java/com/heig/atmanager/ProfileActivity.java

```diff
@@ -14,10 +20,46 @@
  */
 public class ProfileActivity extends AppCompatActivity {
```

Comment:
We can reuse that, the stat can't be a Activity as we discuss during the last metting